pkgs/development/libraries/libgit2/default.nix

```diff
@@ -46,14 +47,31 @@ stdenv.mkDerivation (finalAttrs: {
     "-DCMAKE_LIBRARY_PATH=${stdenv.cc.libc}/lib"
   ];
 
-  nativeBuildInputs = [ cmake python3 pkg-config ];
+  nativeBuildInputs = [ cmake python3 pkg-config ] ++ lib.optionals (!stdenv.hostPlatform.isWindows) [
+    # Needed for `git apply`; see `prePatch`
+    buildPackages.gitMinimal
```

Comment:
Isn't splicing done here automatically?

Reply:
Yeah, should be. It wasn't the case in the original one.

Reply:
Done.